specification/app/resource-manager/Microsoft.App/stable/2022-03-01/examples/AuthConfigs_CreateOrUpdate.json

```diff
@@ -3,7 +3,7 @@
     "subscriptionId": "651f8027-33e8-4ec4-97b4-f6e9f3dc8744",
     "resourceGroupName": "workerapps-rg-xj",
     "containerAppName": "testcanadacentral",
-    "name": "current",
+    "authConfigName": "current",
```

Comment:
>authConfigName [](http://example.com/codeflow?start=5&length=14)

It is strange that the PR includes this directory. The main branch has these changes, so not sure why they show up in PR:

[https://github.com/Azure/azure-rest-api-specs/blob/main/specification/app/resource-manager/Microsoft.App/stable/2022-03-01/examples/AuthConfigs_CreateOrUpdate.json](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/app/resource-manager/Microsoft.App/stable/2022-03-01/examples/AuthConfigs_CreateOrUpdate.json)